servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ServerParentChannelInitializer.java

```diff
@@ -70,4 +73,34 @@ static void initFrameLogger(final Http2FrameCodecBuilder multiplexCodecBuilder,
                             frameLoggerConfig.logLevel()), frameLoggerConfig.logUserData()));
         }
     }
+
+    static io.netty.handler.codec.http2.Http2Settings toNettySettings(Http2Settings h2Settings) {
+        io.netty.handler.codec.http2.Http2Settings nettySettings = new io.netty.handler.codec.http2.Http2Settings();
```

Comment:
Using the default `io.netty.handler.codec.http2.Http2Settings()` will override the default settings set by Netty: maxHeaderListSize=8192.
Consider taking `multiplexCodecBuilder.initialSettings()` and apply only non-null ST settings on top of it. By using `multiplexCodecBuilder.initialSettings()` instead of  `io.netty.handler.codec.http2.Http2Settings.defaultSettings()` you save allocation as the `multiplexCodecBuilder` always invokes `defaultSettings()`.

Reply:
> I removed the empty/default behavior here as defaults are already applied in the builder.

Ok, I saw later that you set all default at ST level. I don't mind either way.

Reply:
Defaults being applied at the builder makes more sense ... then folks can override them and we will use them and/or validate them later.